client/boot/common.js

```diff
@@ -126,18 +129,24 @@ const loggedOutMiddleware = currentUser => {
 	}
 };
 
-const loggedInMiddleware = currentUser => {
+const loggedInMiddleware = ( currentUser, reduxStore ) => {
 	if ( ! currentUser.get() ) {
 		return;
 	}
 
 	page( '/', context => {
 		const { primarySiteSlug } = currentUser.get();
+		const state = reduxStore.getState();
+		const application = getApplication( state );
 		let redirectPath =
 			primarySiteSlug && 'variant' === abtest( 'redirectToCustomerHome' )
 				? `/home/${ primarySiteSlug }`
 				: '/read';
 
+		if ( 'jetpack-cloud' === application ) {
+			redirectPath = '/app';
```

Comment:
Why is `jetpack-cloud` responding to `/app`? It seems to generic to use that one. 

Also, last time I checked, we wanted to use the root of Calypso for the Jetpack Cloud dashboard, and have all of its main sections be specified as direct first-level children (`/scan`, `/backup`).